nautobot/core/graphql/schema.py

```diff
@@ -460,6 +462,15 @@ def already_present(model):
                 # Find the model class based on the content type
                 ct = ContentType.objects.get(app_label=app_name, model=model_name)
                 model = ct.model_class()
+                if model == ComputedField:
+                    for i, cpf in enumerate(ComputedField.objects.all()):
+                        try:
+                            check_if_key_is_graphql_safe("Computed Field", cpf.key)
+                        except ValidationError:
+                            cpf.key = cpf.key.replace(" ", "_")
+                            cpf.key = cpf.key.replace("-", "_")
+                            cpf.key = f"a{i}" + cpf.key
+                            cpf.save()
```

Comment:
Risk of collisions with existing computed fields here when doing replacements? Probably would be better to skip over such fields instead of trying to fix them up.

Also, this seems like a weird place to do this check. I would have thought it'd be something we can/should do in `extend_schema_type_computed_field()` and similar functions.